modules/op-resolver-hub.adoc

```diff
@@ -0,0 +1,138 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-hub-specify_{context}"]
+= Specifying a remote pipeline or task using the Hub resolver
+
+When creating a pipeline run, you can specify a remote pipeline from {artifact-hub} or {tekton-hub}. When creating a pipeline or a task run, you can specify a remote task from {artifact-hub} or {tekton-hub}.
+
+.Procedure
+
+* To specify a remote pipeline or task from {artifact-hub} or {tekton-hub}, use the following reference format in the `pipelineRef` or `taskRef` spec:
++
+[source,yaml]
+----
+# ...
+  resolver: hub
+  params:
+  - name: catalog
+    value: <catalog>
+  - name: type
+    value: <catalog-type>
+  - name: kind
+    value: [pipeline|task]
+  - name: name
+    value: <resource-name>
+  - name: version
+    value: <resource-version>
```

Comment:
```suggestion
    value: <catalog>
  - name: type
    value: <catalog_type>
  - name: kind
    value: [pipeline|task]
  - name: name
    value: <resource_name>
  - name: version
    value: <resource_version>
```
see guidelines on replaceable values: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

![image](https://github.com/openshift/openshift-docs/assets/40172997/854ee5c1-9659-4bcc-bbf4-5ba2556686bb)